### PR TITLE
Added pow(a,b)=a^b function to find power.

### PR DESCRIPTION
**Pull request by anu231bhav**
This function returns a pointer of largeInt object in which big integer answer is stored.
#24

### CODE REVIEW

**Comment by ayaankhan98** [comment]
@anu231bhav tests are missing, please add tests

**Comment by anu231bhav** [comment]
how can I add tests 
help me I am contributing on GitHub for the first time

**Comment by ayaankhan98** [comment]
you can write tests in `tests/operations/pow.cpp`

File: tests/bigint.hpp [hunk 1]
```diff
@@ -0,0 +1,146 @@
+/**
```
Comment (ayaankhan98) [inline tests/bigint.hpp:1]: there is no need to write this file again and again you can just include it direclty in your code like
```
#include <bigint.hpp>
```
please see the sample_test example in `test/operations/`

File: tests/operations/pow.cpp [hunk 2]
```diff
@@ -0,0 +1,88 @@
...
+        return tempo;
+
+}
+
+
+
+
+int main(){
+
+    #ifndef ONLINE_JUDGE
```
Comment (ayaankhan98) [inline tests/operations/pow.cpp]: not required we are not dealing with any online judge

File: tests/operations/pow.cpp [hunk 3]
```diff
@@ -0,0 +1,88 @@
+#include <stdio.h>
+#include <bits/stdc++.h>
```
Comment (ayaankhan98) [inline tests/operations/pow.cpp:2]: don't use bits/stdc++.h

File: tests/operations/pow.cpp [hunk 4]
```diff
@@ -0,0 +1,88 @@
...
+#include<vector>
+using namespace std;
```
Comment (ayaankhan98) [inline tests/operations/pow.cpp:4]: use `std::` instead of `using` directive

**Comment by ayaankhan98** [comment]
@anu231bhav please remove the CMake cache files from the PR

**Comment by ayaankhan98** [comment]
Codacy Here is an overview of what got changed by this pull request:


```diff

Issues
======
- Added 8
           

Clones added
============
- tests/bigint.hpp  1
         
```


See the complete overview on [Codacy](https://app.codacy.com/gh/ayaankhan98/bigINT/pullRequest?prid=6285287&bid=20451766)

File: tests/bigint.hpp [hunk 5]
```diff
@@ -0,0 +1,146 @@
...
+  class largeInt {
+    private:
+      bool sign;
+      std::string number;
+
+    public:
+      largeInt() {
+      }
+
+      largeInt(int) {
```
Comment (ayaankhan98) [inline tests/bigint.hpp:44]: Codacy found an issue: [Member variable 'largeInt::sign' is not initialized in the constructor.](https://app.codacy.com/gh/ayaankhan98/bigINT/pullRequest?prid=6285287)

File: tests/bigint.hpp [hunk 6]
```diff
@@ -0,0 +1,146 @@
...
+      }
+      largeInt(int64_t) {
```
Comment (ayaankhan98) [inline tests/bigint.hpp:46]: Codacy found an issue: [Member variable 'largeInt::sign' is not initialized in the constructor.](https://app.codacy.com/gh/ayaankhan98/bigINT/pullRequest?prid=6285287)

File: tests/bigint.hpp [hunk 7]
```diff
@@ -0,0 +1,146 @@
...
+      }
+      largeInt(long long) {
+      }
+      largeInt(std::string) {
```
Comment (ayaankhan98) [inline tests/bigint.hpp:50]: Codacy found an issue: [Class 'largeInt' has a constructor with 1 argument that is not explicit.](https://app.codacy.com/gh/ayaankhan98/bigINT/pullRequest?prid=6285287)

File: tests/bigint.hpp [hunk 8]
```diff
@@ -0,0 +1,146 @@
...
+
+    public:
+      largeInt() {
+      }
+
+      largeInt(int) {
+      }
+      largeInt(int64_t) {
+      }
+      largeInt(long long) {
```
Comment (ayaankhan98) [inline tests/bigint.hpp:48]: Codacy found an issue: [Member variable 'largeInt::sign' is not initialized in the constructor.](https://app.codacy.com/gh/ayaankhan98/bigINT/pullRequest?prid=6285287)

Comment (ayaankhan98) [inline tests/bigint.hpp:48]: Codacy found an issue: [Class 'largeInt' has a constructor with 1 argument that is not explicit.](https://app.codacy.com/gh/ayaankhan98/bigINT/pullRequest?prid=6285287)

File: tests/bigint.hpp (hunk 6 above)
Comment (ayaankhan98) [inline tests/bigint.hpp:46]: Codacy found an issue: [Class 'largeInt' has a constructor with 1 argument that is not explicit.](https://app.codacy.com/gh/ayaankhan98/bigINT/pullRequest?prid=6285287)

File: tests/bigint.hpp (hunk 5 above)
Comment (ayaankhan98) [inline tests/bigint.hpp:44]: Codacy found an issue: [Class 'largeInt' has a constructor with 1 argument that is not explicit.](https://app.codacy.com/gh/ayaankhan98/bigINT/pullRequest?prid=6285287)

File: tests/bigint.hpp (hunk 7 above)
Comment (ayaankhan98) [inline tests/bigint.hpp:50]: Codacy found an issue: [Member variable 'largeInt::sign' is not initialized in the constructor.](https://app.codacy.com/gh/ayaankhan98/bigINT/pullRequest?prid=6285287)

**Comment by ayaankhan98** [comment]
Codacy Here is an overview of what got changed by this pull request:


```diff

Issues
======
- Added 8
           

Clones added
============
- tests/bigint.hpp  1
         
```


See the complete overview on [Codacy](https://app.codacy.com/gh/ayaankhan98/bigINT/pullRequest?prid=6285287&bid=20451766)

File: tests/bigint.hpp (hunk 5 above)
Comment (ayaankhan98) [inline tests/bigint.hpp:44]: Codacy found an issue: [Member variable 'largeInt::sign' is not initialized in the constructor.](https://app.codacy.com/gh/ayaankhan98/bigINT/pullRequest?prid=6285287)

File: tests/bigint.hpp (hunk 7 above)
Comment (ayaankhan98) [inline tests/bigint.hpp:50]: Codacy found an issue: [Member variable 'largeInt::sign' is not initialized in the constructor.](https://app.codacy.com/gh/ayaankhan98/bigINT/pullRequest?prid=6285287)

File: tests/bigint.hpp (hunk 8 above)
Comment (ayaankhan98) [inline tests/bigint.hpp:48]: Codacy found an issue: [Member variable 'largeInt::sign' is not initialized in the constructor.](https://app.codacy.com/gh/ayaankhan98/bigINT/pullRequest?prid=6285287)

Comment (ayaankhan98) [inline tests/bigint.hpp:48]: Codacy found an issue: [Class 'largeInt' has a constructor with 1 argument that is not explicit.](https://app.codacy.com/gh/ayaankhan98/bigINT/pullRequest?prid=6285287)

File: tests/bigint.hpp (hunk 5 above)
Comment (ayaankhan98) [inline tests/bigint.hpp:44]: Codacy found an issue: [Class 'largeInt' has a constructor with 1 argument that is not explicit.](https://app.codacy.com/gh/ayaankhan98/bigINT/pullRequest?prid=6285287)

File: tests/bigint.hpp (hunk 6 above)
Comment (ayaankhan98) [inline tests/bigint.hpp:46]: Codacy found an issue: [Member variable 'largeInt::sign' is not initialized in the constructor.](https://app.codacy.com/gh/ayaankhan98/bigINT/pullRequest?prid=6285287)

File: tests/bigint.hpp (hunk 7 above)
Comment (ayaankhan98) [inline tests/bigint.hpp:50]: Codacy found an issue: [Class 'largeInt' has a constructor with 1 argument that is not explicit.](https://app.codacy.com/gh/ayaankhan98/bigINT/pullRequest?prid=6285287)

File: tests/bigint.hpp (hunk 6 above)
Comment (ayaankhan98) [inline tests/bigint.hpp:46]: Codacy found an issue: [Class 'largeInt' has a constructor with 1 argument that is not explicit.](https://app.codacy.com/gh/ayaankhan98/bigINT/pullRequest?prid=6285287)

**Pull request closed without merging**